test/release/examples/benchmarks/shootout/fasta.chpl

```diff
@@ -133,7 +133,7 @@ proc randomMake(desc, a, n) {
         line_buff[i] = a[hi](nucl);
       }
     }
-    line_buff[bytes] = newline;
+    line_buff[bytes] = newline:int(8);
```

Comment:
Looks like I didn't reply quite quick enough to catch you.  I re-tried what I tried earlier and verified that my description above is correct.  The challenge to converting all int(8)s to uint(8)s comes in the declaration of ALU:

    const ALU: [...] uint(8) = [G, G, C, ...];

where G, C, etc. are enums.  We have a long-standing bug/feature-request relating to coercions of param uints and enums which I believe this is an instance of (and I believe the need to cast a known-to-fit-in-an-int(8) param uint(8) value is related as well).

Since the intent of my question was to ask whether you objected to ascii() returning a uint(8) rather than int(8) and you didn't seem to have a problem with that part of the PR, I'm going to merge in order to get some testing going sooner.  We can continue to come up with the ideal expression of fasta after the fact if there's something I'm missing.

(I'm also hoping to look into the coercion issues above again once I get free of another few PRs).